health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -0,0 +1,352 @@
+                            setactionFieldsToRight
+                            sortActionFields
+                            style={{}}
+                        />
+                    </div>
+                }
+                {showToast && <Toast label={showToast.label} type={showToast.isError} onClose={() => setShowToast(null)} />}
+                {dataCreateToast && <Toast label={t("DATA_CREATION_IN_PROGRESS")} type={"info"} transitionTime={600000000} />}
+                {previewPage && (
+                    <Card type={"primary"} variant={"viewcard"} className={"example-view-card"}>
+                        <div style={{fontSize:"2.5rem", fontWeight:700}}>{t("CONFIRM_BOUNDARY_DATA")}</div>
+                        <div style={{height:"1.5rem"}}></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)